ports/glutin/window.rs

```diff
@@ -259,7 +261,11 @@ impl Window {
     #[cfg(not(target_os="linux"))]
     fn handle_next_event(&self) -> bool {
         let event = self.window.wait_events().next().unwrap();
-        self.handle_window_event(event)
+        let mut close = self.handle_window_event(event);
+        while let Some(event) = self.window.poll_events().next() {
+            close = self.handle_window_event(event) || close;
```

Comment:
I think this changes behaviour slightly - since we'll now handle events that are posted after the close event. Should we break out of the loop once we get the close event?